add gomock to tools target and readme section (#22)
openshift/pagerduty-short-circuiter: README.md

### Option 1: Install binary

> This command will fetch, build a binary named `pdcli` and install them to your $GOPATH/bin, you should and move this binary onto your $PATH if desired.

Reviewer: ```suggestion
Build and copy `pdcli` to your $GOPATH/bin:
```
